urban/src/main.cpp

```diff
@@ -69,6 +65,6 @@ void loop() {
 
 void onSerialData()
 {
-    ecu.onSerialData();
+    ecu.handle();
```

Comment:
It's not related to this PR but I just noticed we have the ECU handlers in the urban code. I suppose it was left over from testing but that definitely shouldn't be there 😛. Good catch on removing the RSSI stuff so this'll compile, though.

Reply:
Fixed!